zombies.py

```diff
@@ -19,10 +19,10 @@ def d_zombie(t, y, b, m, alpha, z, k, r):
     """
     S, Z, U, D = y
     result = [
-        (b - m - alpha*Z)*S + r*U,
-        z*U - k*S*Z,
-        alpha*S*Z - z*U,
-        (k*Z+m)*S
+        (b - m - alpha*Z)*S + r*U, #dS
```

Comment:
dS is losing those who are bitten (alpha*S*Z) and then adding those who recovered (+ r*U). (This is the arrow that would be pointing from Undead to Susceptible)